project.clj

```diff
@@ -10,12 +10,12 @@
                  [ring/ring-defaults "0.2.1"]
                  [ring/ring-json "0.4.0"]
                  [ring/ring-jetty-adapter "1.5.0"]
-                 [environ "1.0.0"]]
+                 [environ "1.1.0"]]
   :min-lein-version "2.0.0"
   :plugins [[lein-ring "0.9.7"]
-            [environ/environ.lein "0.3.1"]]
-  :hooks [environ.leiningen.hooks]
+            [lein-environ "1.1.0"]]
+  ;:hooks [environ.leiningen.hooks]
   :ring {:handler facebook-example.core/app}
-  :uberjar-name "facebook-example-standalone.jar"
-  :profiles {:default [:base :dev :user]
-             #_:production #_{:env {:production false}}})
+  :uberjar-name "facebook-example-standalone.jar")
+  ; :profiles {:default [:base :dev :user]
+  ;            #_:production #_{:env {:production false}}})
```

Comment:
`:profiles` in Heroku's clojure example app:
`:profiles {:production {:env {:production true}}}`